lib/Rex/Commands/Iptables.pm

```diff
@@ -338,13 +372,28 @@ sub is_nat_gateway {
 
     my @iptables_option = ();
 
-    my ($default_line) = run "/sbin/ip r |grep ^default";
+    my ($default_line) = run "/sbin/ip $ip_version r |grep ^default";
     my ($dev)          = ( $default_line =~ m/dev ([a-z0-9]+)/i );
     Rex::Logger::debug("Default GW Device is $dev");
 
-    sysctl "net.ipv4.ip_forward" => 1;
-    iptables t => "nat", A => "POSTROUTING", o => $dev, j => "MASQUERADE";
-
+    if ( $ip_version == -6 ) {
+      sysctl "net.ipv6.conf.all.forwarding",     1;
+      sysctl "net.ipv6.conf.default.forwarding", 1;
```

Comment:
Would it make sense to wrap these `sysctl` commands into the block after the upcoming check for iptables version on the next line (i.e. only execute them if iptables actually supports IPv6 NAT)?